src/components/blog/blog-page.js

```diff
@@ -15,26 +15,41 @@ class Blog extends Component {
 
   render() {
     if (this.props.apiBlogData.loaded) {
-      let MEDIUM_PUBLICATION = this.props.apiInfoData.footerData.mediumSlug
-      return (
-        <Container styleName="common.margin">
-          <Card.Group itemsPerRow={3} stackable doubling>
-            {this.props.apiBlogData.data.map((info, id) => (
-              <BlogDetail info={info} key={id} />
-            ))}
-          </Card.Group>
-          <Segment basic padded textAlign="center">
-            <Icon
-              name="medium"
-              size="large"
-              link={true}
-              onClick={() =>
-                window.open(`${MEDIUM_URL}${MEDIUM_PUBLICATION}`, '_blank')
-              }
-            />
-          </Segment>
-        </Container>
-      )
+      if (this.props.apiBlogData.data && this.props.apiBlogData.data.length) {
+        let MEDIUM_PUBLICATION = this.props.apiInfoData.footerData.mediumSlug
+
+        return (
+          <Container styleName="common.margin">
+            <Card.Group itemsPerRow={3} stackable doubling>
+              {this.props.apiBlogData.data.map((info, id) => (
+                <BlogDetail info={info} key={id} />
+              ))}
+            </Card.Group>
+            <Segment basic padded textAlign="center">
+              {MEDIUM_PUBLICATION && MEDIUM_PUBLICATION !== '' ? (
```

Comment:
In my opinion, if the data in the blog list is there, `MEDIUM_PUBLICATION` **won't** be an empty string. Maybe we can get rid of this condition.

Reply:
In the backend the field is Blank = True. So technically it can be an empty string.

Reply:
But if it's `""`, `this.props.apiBlogData.data.length` will be equal to `0`, right?

Reply:
yes

Reply:
So it won't enter this condition anyway